internal/tracinginterceptor/interceptor.go

```diff
@@ -230,3 +288,13 @@ func getPropagationCarrier(headers map[string]string, transport string) Propagat
 	}
 	return opentracing.TextMapCarrier(headers)
 }
+
+func wrapTracedClientStream(tracedStream *tracedClientStream) *transport.ClientStream {
+	wrapped, err := transport.NewClientStream(tracedStream)
+	if err != nil {
+		// This should not happen, as NewClientStream only fails for the nil streams.
+		tracedStream.span.LogFields(logFieldEventError, log.String("message", "Failed to wrap traced client stream"))
+		return tracedStream.clientStream
```

Comment:
Even if this failure mode is not likely to happen, let's close the span before the return, otherwise span will not be closed.